### PR TITLE
REGION_REDIRECT clarification

### PR DESCRIPTION
**Pull request by b1tninja**

### CODE REVIEW

File: keepercommander/loginv3.py [hunk 1]
```diff
@@ -110,20 +111,29 @@ def login(params: KeeperParams):
                 raise Exception('This account need to be created.' % rest_api.CLIENT_VERSION)
 
             elif resp.loginState == proto.REGION_REDIRECT:
+                p = urlparse(params.server)
+                new_server = urlunparse((p.scheme, resp.stateSpecificValue, '/api/rest/', None, None, None))
```
Comment (maksimu) [inline keepercommander/loginv3.py]: We don't really need `/api/rest/` in the url. config.json can have just `https://keepersecurity.[com|eu|com.au]`

**Comment by maksimu** [comment]
Also need to document in the readme list of all available regions:

| Region  | URL                           | Property in `config.json` file              |
|---------|-------------------------------|---------------------------------------------|
| US      | https://keepersecurity.com    | [none]                                      |
| EU      | https://keepersecurity.eu     | `"server": "https://keepersecurity.eu"`     |
| AU      | https://keepersecurity.com.au | `"server": "https://keepersecurity.com.au"` |

File: README.md [hunk 2]
```diff
@@ -142,20 +142,21 @@ optional arguments:
...
-For customers hosted in the US region, set the US endpoint:
-```shell script
-{
-  "server": "https://keepersecurity.com/api/v2/"
-}
-```
+Commander defaults to the US region, so customers hosted in other regions will need to specify a server in the config.
 
-For customers hosted in the EU region, set the EU endpoint:
+####Example config.json for EU region
```
Comment (maksimu) [inline README.md]: Does not look correct in the rendered version
image

File: README.md [hunk 3]
```diff
@@ -142,20 +142,21 @@ optional arguments:
...
 ```shell script
 {
-  "server": "https://keepersecurity.eu/api/v2/"
+  "server": "https://keepersecurity.eu"
 }
 ```
 
+| Region  | URL                           | Property in `config.json` file              |
+|---------|-------------------------------|---------------------------------------------|
+| US      | https://keepersecurity.com    | ```{"server": "https://keepersecurity.com"}```    |
```
Comment (maksimu) [inline README.md]: How about also specifying that for us it is optional to have this in the config

Comment (b1tninja) [reply README.md]: I did mention it was the default, and that only users of other regions need to set it. I don't think it's useful information to further say it's optional here

Comment (maksimu) [reply README.md]: Ok, that's fine

File: README.md [hunk 4]
```diff
@@ -142,20 +142,22 @@ optional arguments:
...
-For customers hosted in the EU region, set the EU endpoint:
+#### Example config.json for EU region
 ```shell script
 {
-  "server": "https://keepersecurity.eu/api/v2/"
+  "server": "https://keepersecurity.eu"
 }
 ```
 
+| Region  | URL                           | Property in `config.json` file              |
```
Comment (maksimu) [inline README.md]: Add a header for this table or move it above "Example config.json for EU region" section, between lines 145 and 146

**Pull request merged**